packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset1.js

```diff
@@ -0,0 +1,43 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import { loadTimelinePopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const bobRossIpsum = "Just use the old one inch brush. It's so important to do something every day that will make you happy. "
+  + "I sincerely wish for you every possible joy life could bring. Water's like me. It's laaazy ... Boy, it "
+  + "always looks for the easiest way to do things\n\nWe're trying to teach you a technique here and how to use it. "
+  + 'The least little bit can do so much. There we go.\n\nNo worries. No cares. Just float and wait for the wind to blow you around. '
+  + "These trees are so much fun. I get started on them and I have a hard time stopping. We'll make some happy little bushes here. "
+  + 'Absolutely no pressure. You are just a whisper floating across a mountain. There are no limits in this world. '
+  + 'A thin paint will stick to a thick paint.\n';
```

Comment:
same nit: use double quotes for all lines

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/231/commits/be1e593d65b150312127d5698a853d9f70ec4979).